shuup/core/models/_orders.py

```diff
@@ -1092,5 +1092,33 @@ def get_customer_name(self):
             if getattr(self, "%s_id" % attr):
                 return getattr(self, attr).name
 
+    def get_available_shipping_methods(self):
+        """
+        Get available shipping methods.
+
+        :rtype: list[ShippingMethod]
+        """
+        from shuup.core.models import ShippingMethod
```

Comment:
Is this imports here really necessary?

Reply:
Importing these in "header" causes circular import (using `shuup.core.models`). Direct import might be possible though but it's ugly :)